Fix support for xarray.DataArray in transform.rescale (#7159)
scikit-image/scikit-image: doc/source/user_guide/numpy_images.rst

    >>> type(camera)
    <type 'numpy.ndarray'>

.. note::

>     Complex array-like types, such as

Reviewer: ```suggestion
    Rich array-like data types, such as
```
Hesitating between "Rich" and "Labeled" (`pandas` and `xarray` mention 'labeled' first and foremost).

Author: "Labeled" sounds great, thanks for the suggestion!

    `pandas.DataFrame <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html>`_
    or `xarray.DataArray <https://docs.xarray.dev/en/stable/generated/xarray.DataArray.html>`_,
    are generally not supported in ``scikit-image``. However, data stored in
    these types can be converted to ndarray with certain assumptions
    (see ``pandas.DataFrame.to_numpy()`` and ``xarray.DataArray.data``). Particularly,

>     such conversion ignores the sampling coordinates (``DataFrame.index``,

Reviewer: ```suggestion
    these conversions ignore the sampling coordinates (``DataFrame.index``,
```

>     ``DataFrame.columns``, or ``DataArray.coords``), which may result in a

Reviewer: ```suggestion
    ``DataFrame.columns``, or ``DataArray.coords``), which may result in
```

>     misrepresented data, for instance, when the original data is irregularly spaced.

Reviewer: ```suggestion
    misrepresented data, for instance, when the original data points are irregularly
    spaced.
```

    >>> type(camera)
    <type 'numpy.ndarray'>

.. note::
    Labeled array-like data types, such as
    `pandas.DataFrame <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html>`_
    or `xarray.DataArray <https://docs.xarray.dev/en/stable/generated/xarray.DataArray.html>`_,
    are not natively supported in ``scikit-image``. However, data stored in these types
    can be converted to ``numpy.ndarray`` with certain assumptions
    (see ``pandas.DataFrame.to_numpy()`` and ``xarray.DataArray.data``). Particularly,

>     these conversion ignore the sampling coordinates (``DataFrame.index``,

Reviewer: ```suggestion
    these conversions ignore the sampling coordinates (``DataFrame.index``,
```
